outlierdetect/outlierdetect.py

```diff
@@ -105,7 +105,8 @@ def compute_outlier_scores(self, frequencies):
                     (number of times aggregation unit reported value).
 
             Returns:
-                dictionary mapping (aggregation unit) -> (MMA outlier score for aggregation unit).
+                dictionary mapping (aggregation unit) -> (MMA outlier score for aggregation unit)
+                dictionaty of value count for each agg_unit
```

Comment:
Let's try to make this a bit more clear? e.g., put the format of the dictionary? I don't think it reads as a clear comment yet

Reply:
I'm trying to match the format of comments that Ben already has... just tried another small iteration. Let me know if that looks better.